client/services/FeedbackGeneratorService.js

```diff
@@ -208,11 +207,42 @@ tie.factory('FeedbackGeneratorService', [
           }
         }
       );
-      feedback.appendCodeParagraph(fixedErrorString);
+
+      // TODO: Will need to adjust depending on the language
```

Comment:
Do you still need this TODO? If so it should be formatted as `TODO(yourusername): ...`